sdk_lib/make_chroot.sh

```diff
@@ -47,6 +47,11 @@ DEFINE_string stage3_path "" \
   "Use the stage3 located on this path."
 DEFINE_string cache_dir "" "Directory to store caches within."
 
+DEFINE_string extra_portdir_overlay_path "" \
+  "Extra PORTDIR_OVERLAY direcory path outside of the chroot."
+DEFINE_string extra_portdir_overlay_name "" \
```

Comment:
Well, assuming src/third_party may be a bit much but expecting a full path as an argument and then enforcing that that path is inside of the source tree should be reasonable. Otherwise you will need to do extra steps like bind mounting the path into the chroot.